sky/clouds/os_image/README.md

```diff
@@ -0,0 +1,14 @@
+# Custom OS Image Generation for SkyPilot
+
+## GCP
+The following is an example of generating a custom skypilot image for GPU instance running Debian 11 OS.
+1. Use SkyPilot to launch a VM \
+`sky launch sky/clouds/os_image/skypilot-gcp-gpu-debian-11.yaml`
+
+2. Stop the VM and create a custom image with name *skypilot-gcp-gpu-debian-11-v{YYMMDD}* \
```

Comment:
Should we use `sky stop` to stop the cluster?

Reply:
Added